datalad_registry/tasks.py

```diff
@@ -117,6 +119,7 @@ def _extract_annex_info(repo) -> InfoType:
 
 
 @celery.task
+@validate_arguments
```

Comment:
why do we need `@validate_arguments` here? or just for general check/validation and not really "conversion" like in the case of tuple-vs-list in `collect_dataset_info`?

Reply:
It does the conversion as well. However, for just converting list to tuple, we don't really need this. I chose this solution because it is terse and looks more uniform with other tasks that we may implement in the future that pass arguments of a Pydantic model type. (Passing arguments of a Pydantic model types to a Celery task is enabled by https://github.com/datalad/datalad-registry/pull/82)